### DISCUSSION TITLE
Meshcentral Desktop Access for a VPS

### DISCUSSION THREAD

**Discussion by rijulsudhir** [Ylianst/MeshCentral#6411] [Q&A]
How can I setup a VPS for Desktop access, when the VPS(Ubuntu 24.04.1 LTS) itself doesn't have a physical display. I tried creating dummy display. In the end all I get is a black display on Meshcentral Website. Please help me. Thank you.

**Comment by si458** [comment]
You need to setup a headless server with a desktop environment then install the meshagent

You might also be able to installvnc and use the web-vnc

Few links to try here

https://www.server-world.info/en/note?os=Ubuntu_24.04&p=desktop&f=1

https://bash-prompt.net/guides/vnc-server/
